resources/definitions/fdmprinter.def.json

```diff
@@ -4775,6 +4775,35 @@
                     "settable_per_mesh": true,
                     "settable_per_extruder": true
                 },
+                "support_z_seam_away_from_model":
+                {
+                    "label": "Support Z Seam Away from Model",
+                    "description": "Manage the spatial relationship between the z seam of the support structure and the actual 3D model. This control is crucial as it allows users to ensure the seamless removal of support structures post-printing, without inflicting damage or leaving marks on the printed model.",
+                    "type": "bool",
+                    "default_value": true,
+                    "enabled": "support_enable",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": true,
+                    "settable_per_meshgroup": false,
+                    "children":
+                    {
+                        "support_z_seam_min_distance":
+                        {
+                            "label": "Min Z Seam Distance from Model",
+                            "description": "The distance between the model and its support structure at the z-axis seam.",
+                            "unit": "mm",
+                            "minimum_value": "0.001",
+                            "value": "line_width * 2",
+                            "default_value": 0.8,
+                            "maximum_value": "line_width * 4",
```

Comment:
I wouldn't add a max value here, we block the user from slicing if the value is set too high. But there is nothing dangerous with actually slicing with such a value, it won't damage the printer or anything. I think just a value warning suffices here.

Reply:
If it doesn't work for "large" values it will also break for "small" values for some weird support structures. The code should be robust against that.

Reply:
It just doesn't find the seam location in that case, with not working thats what I meant.

Reply:
However I understand removing the max value, as the slice can still happen (y)

Reply:
> It just doesn't find the seam location in that case, with not working thats what I meant. It just place the seam at original location. The shortest distance

Hense the value warning

Reply:
Done here: https://github.com/Ultimaker/Cura/pull/18916/commits/f3172cb10ba55f025f1f3bd9b2de5b35c19b0169